plugins/inputs/influxdb_v2_listener/influxdb_v2_listener.go

```diff
@@ -103,6 +104,7 @@ func (h *InfluxDBV2Listener) Init() error {
 	h.readysServed = selfstat.Register("influxdb_v2_listener", "readys_served", tags)
 	h.requestsRecv = selfstat.Register("influxdb_v2_listener", "requests_received", tags)
 	h.notFoundsServed = selfstat.Register("influxdb_v2_listener", "not_founds_served", tags)
+	h.pingServed = selfstat.Register("influxdb_v2_listener", "ping_served", tags)
```

Comment:
To keep naming conventions the same:
```suggestion
	h.pingsServed = selfstat.Register("influxdb_v2_listener", "pings_served", tags)
```